lochness/utils/path_checker.py

```diff
@@ -235,6 +235,32 @@ def update_by_removing_unused_files(df: pd.DataFrame) -> None:
     ds_store_index = df[df.file_name == '.DS_Store'].index
     df.drop(ds_store_index, inplace=True)
 
+    # recording.conf files
+    recording_conf_index = df[df.file_name == 'recording.conf'].index
+    df.drop(recording_conf_index, inplace=True)
+
+    # chat.txt files
+    chat_txt_index = df[df.file_name == 'chat.txt'].index
+    df.drop(chat_txt_index, inplace=True)
+
+
+def update_skipped_av_files(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Update the 'file_check' column in the DataFrame based on the 'parent_dir' column.
+
+    Args:
+        df (pd.DataFrame): The DataFrame containing the data.
+
+    Returns:
+        pd.DataFrame: The updated DataFrame.
+    """
+    for idx, row in df.iterrows():
+        if "Extraneous files" in row["parent_dir"]:
+            df.loc[idx, "file_check"] = True
+        elif "Additional interview files" in row["parent_dir"]:
```

Comment:
What is the guarantee that all subjects, sites will name them as `Extraneous files` , `Additional interview files`. Did you observe the same names across all subjects, sites collected so far?